CHANGELOG.md

```diff
@@ -28,13 +28,16 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - `tt pack`: systemd unit parameterizing support.
 - `tt replicaset vshard`: module to manage vshard in the tarantool replicaset.
   * `tt replicaset vshard bootstrap`: command to bootstrap vshard.
+- `tt connect`: support format for Tarantool tuples for a Tarantool
```

Comment:
move changelog entries to unreleased section

Reply:
Oops, updated.